### ISSUE TITLE
it'll be the best if we have dark theme

### ISSUE THREAD

**Issue by khuctrongoc** [ZhangJian1713/vscode-image-viewer#6]
it'll be the best if we have dark theme

**Comment by ZhangJian1713** [comment]
Yes, it is good, but I don't think it is a high priority. Please be patient and wait. Looking forward to anyone interested in contributing PR for this.

**Comment by Baboonanza** [comment]
So I had a quick go at this. Ideally it should mirror the VSCode theme which can be done using CSS variables, but this seems quite difficult with the version of antd it currently uses (4.x I think?). I've never used atnd but it looks like 5.x introduces quiet simple theming through ConfigProvider which would make it quite easy I think. Otherwise we can get close by spamming bg/fg color into all the styles by that's not so great.

**Comment by ryanwwest** [comment]
Thanks for the awesome plugin. This would be a great addition.

**Comment by zero-enthusiasm** [comment]
I have re-implemented this extension with a dark theme: https://github.com/zero-enthusiasm/vscode-project-image-browser

**Comment by immiProgrammer** [comment]
plz
